babel.config.js

```diff
@@ -1,6 +1,7 @@
 module.exports = {
   presets: ['@babel/env', '@babel/react'],
   plugins: [
+    '@babel/transform-runtime',
```

Comment:
@jxnblk Maybe you can add a comment in the code to explain the reason behind this.

Reply:
Thank you for the examples :) it makes more sense now, it seems to be good for code reusability so I think we should keep it.